tests/wicked/startandstop/sut/t12_openvpn_tunnel_ifdown.pm

```diff
@@ -0,0 +1,53 @@
+# SUSE's openQA tests
+#
+# Copyright © 2018 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Startandstop test cases for wicked
```

Comment:
Could you define a custom summary for each test module? As an alternative I see to come up with a different approach which would need less repetition of boiler-plate code in each file.

Reply:
sure, will do